interface/patient_file/summary/stats_full.php

```diff
@@ -157,22 +225,40 @@ function newEncounter() {
 
                             // Show header
                             $disptype = $focustitles[0];
+                            echo "<div class='mb-3'><span class='title mr-2'>" . text($disptype) . "</span>\n";
+
                             if (AclMain::aclCheckIssue($focustype, '', array('write', 'addonly'))) {
                                 if (($focustype == 'allergy' || $focustype == 'medication') && $GLOBALS['erx_enable']) {
                                     echo "<a href='../../eRx.php?page=medentry' class='btn btn-primary btn-sm' onclick='top.restoreSession()' >" .
-                                    xlt('Add') . "</a>\n";
+                                        xlt('Add') . "</a>\n";
                                 } else {
-                                    echo "<a href='javascript:;' class='btn btn-primary btn-sm btn-add mr-3' onclick='dopclick(0," .
-                                    attr_js($focustype)  . ")'>" . xlt('Add') . "</a>\n";
+                                    echo "<a href='javascript:;' class='btn btn-primary btn-sm btn-add mr-1' onclick='dopclick(0," .
+                                        attr_js($focustype)  . ")'>" . xlt('Add') . "</a>\n";
                                 }
                             }
 
-                            echo "  <span class='title'>" . text($disptype) . "</span>\n";
-                            echo "<div class='table-responsive'><table class='table mb-3'>";
+                            $canDelete = AclMain::aclCheckCore('admin', 'super');
+                            if ($canDelete) {
+                                echo "<button id='" . $focustype . "-delete' disabled
+                                    class='btn btn-primary btn-sm btn-delete btn-danger mr-1'
+                                    onclick='deleteSelectedIssues(" . attr_js($focustype)  . ")'>" . xlt('Delete') . "</button>\n";
```

Comment:
Hi @tpantelis

In here, the button has a two-color style `btn-primary` and `btn-danger`, but it should only have one color, So please remove the `btn-primary` style.

Reply:
Done